test/powershell/Language/Scripting/ActionPreference.Tests.ps1

```diff
@@ -15,7 +15,7 @@ Describe "Tests for (error, warning, etc) action preference" -Tags "CI" {
             $err = $null
             try
             {
-                get-childitem nosuchfile.nosuchextension -ea stop -ev err
+                get-childitem nosuchfile.nosuchextension -ErrorAction stop -ev err
```

Comment:
`-ev` should be changed to `-ErrorVariable`

Reply:
Fixed.